install/caj/bin/ts.jte

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
should be 2021

Reply:
For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. copyright go copied in the process. I have reverted copyright now.